components/Dialogs/EstimateChangeDialog.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { Text, Modal, Radio } from '@geist-ui/react';
+import { useRouter } from 'next/router';
+import { parseCookies } from 'nookies';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Story, UrlParams } from '../../redux/types';
+import { editStory } from '../../redux/actions/stories.actions';
+import { getApiKey } from '../../redux/selectors/settings.selectors';
+import PivotalHandler from '../../handlers/PivotalHandler';
+import { useAsync } from '../../hooks';
+
+const CenteredDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+interface EstimateChangeDialogParams {
+  story: Story;
+  open: boolean;
+  state: string;
+  onClose: () => void;
+}
+
+const EstimateChangeDialog = ({ story, open, state, onClose }: EstimateChangeDialogParams): JSX.Element => {
+  const dispatch = useDispatch();
+  const apiKey = useSelector(getApiKey);
+  const router = useRouter();
+  const { id }: UrlParams = router.query;
+  const [selectedEstimate, setSelectedEstimate] = useState(story?.estimate ? String(story.estimate) : '');
+
+  const [{ isLoading }, changeEstimate] = useAsync(async () => {
+    const newStory: Story = { ...story, estimate: Number(selectedEstimate) };
+    dispatch(editStory({ projectId: id, story: newStory, storyState: state }));
+    const pivotal = new PivotalHandler();
```

Comment:
Maybe we should use static functions instead of having to instantiate a new class every time we want to use a pivotal method